### ISSUE TITLE
revealjs a11y: line numbers in code cells

### ISSUE THREAD

**Issue by cscheid** [quarto-dev/quarto-cli#13574]
reprex:

````
---
format: revealjs
---

```{python}
#| echo: true
print("hello")
a = 3
```
````

The line numbers are all links with no content. That's a no-no.

**Comment by cwickham** [comment]
Just realized my recently opened #14655 is a duplicate of this. Closing this in favor of that since it has a bit more detail of the actual violation. I'll note the `revealjs` part there.
